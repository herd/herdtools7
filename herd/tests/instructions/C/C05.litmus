C C05
Variant=S128

{
__int128_t x=1;
}

P0 (__int128_t *x) {
 *x = 0;
}

forall ([x]=0)
