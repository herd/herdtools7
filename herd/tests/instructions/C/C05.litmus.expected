Test C05 Required
States 1
[x]=0x0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([x]=0x0)
Observation C05 Always 1 0
Hash=db772c002d727bd43e0889f42282f9f6

